java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/JavaCheckVerifier.java

```diff
@@ -171,20 +196,23 @@ private void collectExpectedIssues(String comment, int line) {
     if (comment.startsWith(TRIGGER)) {
       String cleanedComment = StringUtils.remove(comment, TRIGGER);
 
+      Map<IssueAttribute, String> attr = new HashMap<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Convert this Map to an EnumMap. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1640)